docs/content/preview/yugabyte-platform/configure-yugabyte-platform/on-premises-script.md

```diff
@@ -72,10 +72,10 @@ You can manually provision each node using the pre-provisioning Python script, a
     SUDO password:
     ```
 
-1. Enter your password.
+2. Enter your password.
```

Comment:
You don't need to order the steps as 1,2,3,4. As a markdown convention - https://markdownguide.offshoot.io/basic-syntax/#ordered-lists, we use 1,1,1... as the numbering format. So you can revert the change.

Reply:
Made the changes